.travis.yml

```diff
@@ -31,7 +38,7 @@ matrix:
           env: WP_VERSION=latest WP_MULTISITE=0 WP_CORE_DIR=/tmp/wordpress WC_VERSION=3.8.1
     allow_failures:
         php: 7.3
-        env: WP_VERSION=latest WP_MULTISITE=0 WP_CORE_DIR=/tmp/wordpress RUN_RANDOM=1 COMPOSER_DEV=1
```

Comment:
I couldn’t think of a reason why this config should be allowed to fail, so I replaced it with the `Latest WooCommerce` build config.

Reply:
Decided to revert replacing this so it can be investigated more.